src/vs/workbench/api/common/extHost.api.impl.ts

```diff
@@ -771,30 +771,30 @@ export function createApiFactoryAndRegisterActors(accessor: ServicesAccessor): I
 			}
 		};
 
-		const tasks: typeof vscode.tasks = {
+		const tasks: typeof vscode.tasks = { // {{SQL CARBON EDIT}} disable tasks api
```

Comment:
Hm, do we want to remove these from vscode.d.ts as well so extensions don't try to mistakenly use them?

Reply:
We initially did that for the debug apis but found it was difficult for extensions to then compile for ads. Karl would know more.